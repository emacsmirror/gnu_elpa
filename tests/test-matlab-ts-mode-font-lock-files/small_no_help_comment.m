% -*- mode: matlab-ts -*-
function out = small_no_help_comment(in)

    % code comment
    out = in;
end
