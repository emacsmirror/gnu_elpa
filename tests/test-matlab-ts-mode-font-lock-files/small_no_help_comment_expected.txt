c ccc ccccc ccccccccc ccc
kkkkkkkk vvv d fffffffffffffffffffffbvvb

    c cccc ccccccc
    ddd d ddD
kkk
